src/generators/azuredevops/generator/ExtensionGenerator.ts

```diff
@@ -30,13 +31,19 @@ export default class ExtensionGenerator implements IGenerator<{ publisher: strin
         this.log(`Extension: ${extension.name} is already installed.`);
       }
 
-      return installed;
+      return !installed;
     });
 
     const installPromises: Array<Promise<void> | undefined> = uninstalledEntensions
       .map((extension) => this.installExtension(extension.publisher, extension.name));
 
-    return Promise.all(installPromises);
+    await Promise.all(installPromises);
+
+    if (installPromises.length > 0) {
+      this.log('Waiting 1 minute for the extension(s) to fully install.');
+      const ONE_MINUTE = 1 * 60 * 1000;
+      await sleep(ONE_MINUTE);
```

Comment:
Polling `this.conn.getInstalledExtensions()` doesn't work for this one right?

Reply:
Yeah, we tried that before 😞

Reply:
Ah, I remember that now. Just checking 🙂